ci/jenkins_status.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
```

Comment:
This is probably okay, but some distros point `python -> python2` so personally I tend to use `python3` here

Reply:
I hope we're beyond that now!

Reply:
Hopefully, but some older (still supported) distros still use python2 by default last I checked. Also, some distros don't symlink `python` at all. E.g.
```
$ cat /etc/os-release
NAME="SLES"
VERSION="15-SP4"
VERSION_ID="15.4"
PRETTY_NAME="SUSE Linux Enterprise Server 15 SP4"
ID="sles"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:15:sp4"
DOCUMENTATION_URL="https://documentation.suse.com/"

$ which python
which: no python in ...

$ which python3
/usr/bin/python3
```